javaslang/src/main/java/javaslang/collection/PriorityQueue.java

```diff
@@ -558,8 +571,8 @@ private Node(T root, int rank, Seq<Node<T>> children) {
                 assert rank == tree.rank;
 
                 return comparator.isLessOrEqual(this.root, tree.root)
-                       ? of(this.root, this.rank + 1, tree.appendTo(this.children))
-                       : of(tree.root, tree.rank + 1, this.appendTo(tree.children));
+                        ? of(this.root, this.rank + 1, tree.appendTo(this.children))
```

Comment:
Looks like number of spaces % 4 is 0. It is likely the IDE changed indentation. For me it is ok...